### PR TITLE
Create Github build actions for CI

### PR DESCRIPTION
**Pull request by akondas**

### CODE REVIEW

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      - uses: actions/checkout@v1
+
+      - name: Validate composer.json and composer.lock
+        run: composer validate
+
+      - name: Install dependencies
+        run: composer install
+
+     - name: Run test suite
+       run: composer check
```
Comment (akondas) [inline .github/workflows/php.yml]: add cach :warning:

**Comment by akondas** [comment]
:thinking: @github why actions are not running on new pull requests?

**Comment by marmichalski** [comment]
Why keeping travis tho?

**Comment by akondas** [comment]
Yup, say good bye to Travis :clap:

**Pull request merged**